src/components/modal/index.js

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useState } from 'react';
+import useTranslation from 'hooks/useTranslation';
+import { z } from 'zod';
+import { useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+
+import Button from 'components/common/Button';
+import smileIcon from 'assets/smilies.svg';
+import Input from 'components/form/Input';
+import TextArea from 'components/form/textarea';
+import { useQuestionMutation } from 'services/questions/questions';
+import close from 'assets/close.svg';
+
+import './styles.scss';
+
+const Modal = ({ handleCloseModal = null }) => {
```

Comment:
We can change this name to something that reflect the fact this is a Contact form, what do you think? Somethink like `ContactFormModal` maybe?